README.md

```diff
@@ -51,14 +51,29 @@ Visit http://localhost:3000 to see the app.
 The easiest way to create tasks is through the Rails console:
 
 ```ruby
-# Quick task creation
+# Quick task creation using helper methods
 eddie = Child.find_by(name: "Eddie")
 eddie.add_daily_task("Make bed", :morning)
 eddie.add_weekday_task("Take out trash", "1,3,5", :afternoon)  # MWF
 eddie.add_weekend_task("Clean room", :afternoon)
 
-# Or use the interactive task creator
+# More examples with helper methods
+audrey = Child.find_by(name: "Audrey")
+audrey.add_daily_task("Feed cat", :morning)
+audrey.add_daily_task("Clear dishes", :evening)
+audrey.add_weekday_task("Practice piano", "1,2,3,4,5", :afternoon)  # Weekdays
+
+# Using the interactive task creator
 rake chores:create
```

Comment:
Please provide additional comment explaining how to run a rake task. Is it `kamal shell` then `rake chores:create`?